src/common/sentry.ts

```diff
@@ -21,95 +21,78 @@
 
 import { CaptureConsole, Dedupe, Offline } from "@sentry/integrations";
 import type { Event as SentryEvent } from "@sentry/types";
-import { sentryDsn, sentryDsnIsValid, isProduction } from "./vars";
+import { sentryDsn, isProduction } from "./vars";
 import { UserStore } from "./user-store";
 import logger from "../main/logger";
 
 // https://www.electronjs.org/docs/api/process#processtype-readonly
-const electronProcessType = ["browser", "renderer", "worker"] as const;
+type ElectronProcessType = "browser" | "renderer" | "worker";
+
+const processType = process.type as ElectronProcessType;
```

Comment:
I would avoid type casting at best

Reply:
Okay, even you were too? I guess we don't really care about this type cast.

Reply:
I am not fully sure, but constant assertion seems to be different than normal type assertion (type casting)
https://dev.to/mremanuel/short-and-sweet-typescript-const-assertions-1d84

Reply:
They produce different code. Namely the constant assertion produces an array in the JS which can then be used. Similar to how `enum`'s produce an object in the JS.

However, the previous code was doing a type cast anyway. All type casts in TS are purely a compile time construct even if the type came from a constant assertion.